docs/DeveloperGuide.md

```diff
@@ -170,6 +170,66 @@ This is an example of what the edit feature does at every step to achieve its in
      dependent on `filteredProfileList` and `profileList` of the `ItemListManager` reduces testability and
       maintainability.
 
+### Storage Feature
+
```

Comment:
Do you think this should be part of the design component? 
![image](https://user-images.githubusercontent.com/43701650/96705117-722e7900-13c7-11eb-9208-a1cb0082ad2e.png)

Reply:
Sure?